src/audio/multiband_drc/multiband_drc.c

```diff
@@ -28,6 +28,7 @@
 #include <sof/ut.h>
 #include <user/eq.h>
 #include <user/trace.h>
+#include <stdbool.h>
```

Comment:
I don't see this commit adding any uses of this header, or was it missing before?
To the commit text: sorry, I don't see in the code what is described in the commit text

Reply:
it was a previous version, this needs to be removed